worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/ClipboardFormats.java

```diff
@@ -0,0 +1,69 @@
+package com.sk89q.worldedit.extent.clipboard.io;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public final class ClipboardFormats {
+
+    private static final Map<String, ClipboardFormat> aliasMap = new HashMap<>();
+    private static final List<ClipboardFormat> registeredFormats = new ArrayList<>();
+
+    public static void registerClipboardFormat(ClipboardFormat format) {
+        checkNotNull(format);
+
+        for (String key : format.getAliases()) {
+            aliasMap.put(key.toLowerCase(Locale.ENGLISH), format);
```

Comment:
this gives later registered formats absolute priority, potentially making earlier registered formats inaccessible if aliases overlap...